src/nominatim_db/clicmd/api.py

```diff
@@ -91,13 +92,18 @@ def _get_geometry_output(args: NominatimArgs) -> napi.GeometryFormat:
         raise UsageError(f"Unknown polygon output format '{args.polygon_output}'.") from exp
 
 
-def _get_locales(args: NominatimArgs, default: Optional[str]) -> napi.Locales:
+def _get_locales(args: NominatimArgs, config: Optional[Configuration]) -> napi.Locales:
```

Comment:
config is always handed in, so don't make it Optional and simplify the code below accordingly.